### ISSUE TITLE
Add Data Cursor for Analyzer window

### ISSUE THREAD

**Issue by AlopexXx** [betaflight/blackbox-log-viewer#417]
I think a 1D data cursor for Frequency and 2D data cursor for Frequency vs Throttle would be very usefull to make precise readings. Also incremental buttons for scaling like they are on most scroll bars could be usefull.

**Comment by Hawxxer** [comment]
Yes especially the last part, it's very hard to compare the 2d graphs with the frequency and amplitude sliders, just add a numerical value so I can set two plots to the same scale to compare.

**Comment by haslinghuis** [comment]
@demvlad guess this is implemented now :)

**Comment by demvlad** [comment]
> I think a 1D data cursor for Frequency and 2D data cursor for Frequency vs Throttle

The all are work for the new Power spectral density charts. 

Its are possible to do these for the olde frequencies spectrums. But these frequency spectrums are:
- depends from curves scale (zoom) 
- have not any standard units. 

How the olde freq spectrums work: the fft amplitude of SCALED (zoomed) curves data is limited by 100, and these limited values are showed at the chart. 
Therefore higher amplitude are cut, and values have not physical units. 
For example, if you change curves zoom, the spectrums amplitude value will changed too, for same physical data. But it is not physical... 
The frequencies spectrums vertical slider changes spectrums charts zoom. It is Ok. But log data curves zoom changes spectrums amplitude value and spectrums view too. 
The comparing of spectrums values with different curves zoom has not though. What value is good and what is bad in this case? What is mean 10 or 80, 100 noise value for zoomed curve? Is this good or bad? We need some experience to understand this. 
For example, the filtered gyro in GYRO_SAMPLES debug mode has "bigger" frequency spectrums  noise amplitude then common (not debug) unfilteted gyro field. But it's opposite in really! :) 

I suggest to use power spectral density spectrum (PSD) instead of. It's show the all noise ampltude range in real standard dBm/Hz units. These charts (1D, 2D) have the mouse data cursor what you need. 
Also it's have the different spectrums comparison  tools for 1D curves. 
The standard dBm/Hz units allow to compare different spectrums properly. And we can say that gyro noise -40dBm is good, and -10dBm is not, >=0dBm are terrible.. 

https://betaflight.com/docs/wiki/guides/current/BBE-Power-Spectral-Density-charts#the-power-spectral-density-by-throttle-heat-map-diagram--psd-vs-throttle

**Comment by demvlad** [comment]
IMO, the olde frequency spectrums are deprecated now.

**Comment by demvlad** [comment]
>  Also incremental buttons for scaling like they are on most scroll bars could be usefull.

I've made [some PR already](https://github.com/betaflight/blackbox-log-viewer/pull/844). Will continue this improvement soon. Will use the numberic input with spin buttons instead of sliders. 
We can have more precision control and look at it's current value by using numberic input instead of slider.
